tabuckner/broskichat
# Chat message comp

### Project Changed


**- What I Did**

1. Made the register page look like the login page
2. Setup the 'Chat Message Box'. (Changed the design a bit and added a 'emoji' insertion placeholder
3. Added React Font-Awesome SVG package

**- How I Did It**

1. Copied some of the style from Login to Register file
2. Messed around with Chat/Form/Form.jsx file
3. Also I deleted the Users directory and moved Register in the Component directory
4. Added FontAwesome with Yarn

**- How to Test/Verify It**

1. Pull the `chat-message-comp` branch, start up the app and take a look at the Register page and Chat page to see the changes.


**- An Image to Make Us Giggle (not mandatory but encouraged)**
nada
